docs/develop/worker-performance.mdx

```diff
@@ -36,6 +36,154 @@ The Workflow Cache is created and shared between all the workers. It's designed
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide only partial information.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values.
+Since Tasks remain valid for only a few seconds in Sticky Queues, this inaccuracy becomes less significant as the backlog grows in size.
+:::
+
+
+#### `ApproximateBacklogAge` {#ApproximateBacklogAge}
+
+Returns the approximate age of the oldest Task in the backlog.
+The age is based on the creation time of the Task at the head of the queue.
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide only partial information.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values.
+Since Tasks remain valid for only a few seconds in Sticky Queues, this inaccuracy diminishes over time, especially when the backlog is older than a few seconds.
+
+:::
+
+#### `TasksAddRate` and `TasksDispatchRate` {#TasksAddRate-and-TasksDispatchRate}
+
+Reports the approximate Tasks-per-second added to or dispatched from a Task Queue.
+This rate is averaged over the most recent 30-second time interval.
+The calculations include Tasks that were added to or dispatched from the backlog as well as Tasks that were immediately dispatched and bypassed the backlog (sync-matched).
+
+:::info Metric Accuracy
+
+The actual Task delivery count may be significantly higher than the number reported by these metrics:
+
+- Tasks using Eager dispatch do not pass through Task Queues.
+  Eager dispatch refers to a Temporal feature where Activities can be requested by an SDK using one Workflow Task completion response.
+- A Sticky Task Queue is associated with a dedicated Worker instance.
+  Tasks passed to Sticky Task Queues are not accounted for by these metrics.
+  Normally, only the first Workflow Task of each Workflow is placed on a Workflow Task Queue.
+  Subsequent Tasks are passed to the Sticky Task Queue for performance improvement.
+
+:::
+
+#### `BacklogIncreaseRate` {#BacklogIncreaseRate}
+
+Approximates the **net** Tasks per second added to the backlog, averaged over the most recent 30 seconds.
+This is calculated as:
+
+```
+TasksAddRate - TasksDispatchRate
+```
+
+- Positive values of `X` indicate the backlog is growing by about `X` Tasks per second.
+- Negative values of `X` indicate the backlog is shrinking by about `X` Tasks per second.
+
+:::info Metric Accuracy
+
+While individual `add` and `dispatch` rates may be inaccurate due to Eager and Sticky Task Queues, the `BacklogIncreaseRate` reliably reflects the rate at which the backlog is shrinking or growing for backlogs older than a few seconds.
+
+:::
+
+### Retrieve metrics with the Temporal CLI
```

Comment:
just a question - is it implicit that this is the latest version of the temporal cli?

Reply:
Discussing on-thread. Marking as resolved as I fix.